model/form_submission.ts

```diff
@@ -54,6 +54,7 @@ export interface FormSubmission {
   should_subscribe: boolean;
   /** Whether the author has opted into receving updates about the app. */
   confirmed_get_app_updates: boolean;
+  image_uuid: string | null;
```

Comment:
The storage API seems to work using filepaths, so on second thought just storing an UUID is not sufficient- we need to capture the whole filename we upload under.

Another thing we should definitely consider doing is actually defining the form submission ID on the client instead of letting that get generated on the backend. The reason to do that is so that we can use the form submission IDs as the folders for these uploads, which is preferable to a totally flat file list.

Reply:
In this second approach, we don't even need to generate IDs for the files; we could use the original filename (would need to prevent duplicates within the same submission though).

Reply:
the form_submission table now has a col called image_uuid which is used as a the folder name in the api and relative files are stored in folder with file name and extension